### DISCUSSION TITLE
Compatibility with Craft 1.63?

### DISCUSSION THREAD

**Discussion by feliao** [kudrykv/alfred-craftdocs#7] [Q&A] [answered]
Great workflow and a huge fan of both Alfred and Craft!

Does anyone know if version V0.2.5 of this workflow works with Craft 1.63?

Mine is broken and I get the following messaging in the log? I imagine the data store system/structure has changed in the last releases?

[14:09:17.864] ERROR: Craft Docs[Script Filter] Code 1: cp: /Users/felixliao/Library/Containers/com.lukilabs.lukiapp/Data/Library/Application Support/com.lukilabs.lukiapp/ is a directory (not copied).

**Comment by kudrykv** [comment]
I'm tracking the issue here: #8 

Unfortunately, there is no right away remedy for this. The good news is, the fix would make the workflow work faster. I need a couple of days to get to it, though, so that's the bad news.

**Comment by kudrykv** [comment, marked as answer]
@feliao try the updated version — https://github.com/kudrykv/alfred-craftdocs/releases/download/v0.2.6/Craft_Docs_v0.2.6.alfredworkflow

**Reply by feliao** [reply]
Hi Seems to work now. search seems to take longer now ? Mine seems to take up to 5 seconds to refresh to the correct results list ..

**Reply by kudrykv** [reply]
Have you increased the number of documents recently? That might be the case.

Current implementation didn't change; only paths were updated. So, if anything has changed, the change has been introduced on Craft side.

I'm looking into an alternative to the current implementation to make it faster. I'll keep you posted if you're interested.

**Comment by feliao** [comment]
Yes, my repository is bigger now but not excessive so I thought a 5 seconds delay was a bit strange. Happy to help test things out!

Thanks,
--
Felix Liao

**Reply by kudrykv** [reply]
You may try this one: https://github.com/kudrykv/alfred-craftdocs-searchindex

It is different in how the data is pulled. The current one copies DB over and then connects to it to query. The reason for copying is the fragility of DB versions.

The search-index workflow connects to another database within the Craft ecosystem. It has fewer features and somewhat worse search capabilities, but it should be times faster.

**Comment by feliao** [comment]
Failed !!

     "title": "failed to query database",
      "subtitle": "search: search: Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub",
      "valid": false,
      "icon": {
        "path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertCautionIcon.icns"
      }
    }
  ]
}

Thanks,
--
Felix Liao

**Reply by kudrykv** [reply]
Thank you for testing it out!

I've rebuilt the workflow, please see if it works https://github.com/kudrykv/alfred-craftdocs-searchindex/releases/tag/v0.1.2

**Comment by feliao** [comment]
This one works and is fast! Is there anyone to search across all the spaces (including the non-personal ones).

Thanks,
--
Felix Liao

**Reply by kudrykv** [reply]
That would be a fun thing to tackle. I can't get an ETA for it yet, though.
